src/main/drivers/motor.c

```diff
@@ -321,9 +321,15 @@ timeMs_t motorGetMotorEnableTimeMs(void)
 #endif
 
 #ifdef USE_DSHOT_BITBANG
-bool isDshotBitbangActive(const motorDevConfig_t *motorDevConfig) {
+bool isDshotBitbangActive(const motorDevConfig_t *motorDevConfig)
+{
     return motorDevConfig->useDshotBitbang == DSHOT_BITBANG_ON ||
         (motorDevConfig->useDshotBitbang == DSHOT_BITBANG_AUTO && motorDevConfig->useDshotTelemetry && motorDevConfig->motorPwmProtocol != PWM_TYPE_PROSHOT1000);
 }
 #endif
+
+float getDigitalIdleOffset(const motorConfig_t *motorConfig)
+{
+	return CONVERT_PARAMETER_TO_PERCENT(motorConfig->digitalIdleOffsetValue * 0.01f);
```

Comment:
0.01f or 0.001f ?

Reply:
0.01 - the unit of this is 1 / 10000, i.e. 0.01%: https://github.com/betaflight/betaflight/blob/49679ce0efda8684d1c7129e888dfc091978dfb6/src/main/pg/motor.h#L56